portfolio/src/App.jsx

```diff
@@ -256,16 +243,20 @@ function App() {
             </div>
           </div>
         </section>
-        <section className="py-8" section id="About-me">
+        <section className="py-8" id="about-me">
           <div className="container m-auto px-4">
             <h2 className="text-2xl font semi-bold">About Me</h2>
             <div className="mt-12 relative before:absolute before:top-0 before:left-16 before:rounded-full before:bottom-10 before:w-1 before:bg-white">
               <div className="pl-24 mt-12 relative before:w-4 before:h-4 before:bg-gradient-to-t before:from-orange-500 before:to-yellow-500 before:absolute before:rounded-full before:left-[58px]">
                 <h3 className="absolute -left-0 text-lg font-semibold">2023</h3>
                 <p>
-                  Prior to enrolling into the UTA Coding Bootcamp, I was a general laborer. I have 2 years of customer service experience and
```

Comment:
I noticed this a few places. I definitely don't blame you for this one, but just a heads up that UTA refers to University of Texas at Arlington (which I don't think is the UT that you are referring to, but I could be wrong about that, in which case you can totally ignore me!). I noticed "UTA" a few places on your portfolio site as well as on your resume. I would use the full name "University of Texas at Austin" where applicable. Also, another thing that I noticed on your resume is that you suggested that you were graduating from UT instead of specifying that you were graduating from the UT Austin Coding Bootcamp. Here's another gotcha that will very quickly cause you to get skipped over and/or ignored by a recruiter. Suggesting that you are graduating from UT gives the connotation that you are graduating there with a four year degree like a Bachelors. So I would be very careful there. Just another thing where specificity really matters and in fact could be the difference between you getting a follow-up call from a recruiter or getting completely ignore or even blacklisted from a recruiting agency for incorrect information about yourself.